app.js

```diff
@@ -2,8 +2,15 @@ const express = require("express");
 const app = express();
 
 const{ getTopics } = require("./controllers/topics.controller");
+const endpointController = require('./controllers/endpoints.controller');
 
 app.get("/api/topics", getTopics);
+app.get('/api', endpointController.getApiInfo);
 
+app.use((err, request, response, next) =>{
+    if(err.status===500){
```

Comment:
There's no need for a if statement here. This error handler is the last case if the error doesn't apply to any of your other error handlers